_data/es/navigation.yml

```diff
@@ -25,4 +25,4 @@ nav:
     - page:              Demos
       url:               /wiki/Demos
     - page:              Declaración de Privacidad
-      url:               /wiki/Privacy-Statement
\ No newline at end of file
+      url:               /wiki/Privacy-Statement
```

Comment:
What's this change? I don't see anything here...

Reply:
No idea. I don't recall editing that line.

Reply:
Ok. Strange if it doesn't change anything I think it's ok.